data/data/coreos/rhcos.json

```diff
@@ -346,76 +346,88 @@
               }
             },
             "raw.gz": {
               "disk": {
-                "location": "https://rhcos.mirror.openshift.com/art/storage/prod/streams/4.12/builds/412.86.202212081411-0/s390x/rhcos-412.86.202212081411-0-metal.s390x.raw.gz",
-                "sha256": "b1a317170807216afee7d3ad4c7964569624dc5d28ffe8370eae90f21ac634cc",
-                "uncompressed-sha256": "bb90b5fdf7d804773fdb85265b19e104ae17fb82b0238d3aca92589f2ce63d0c"
+                "location": "https://rhcos.mirror.openshift.com/art/storage/prod/streams/4.12/builds/412.86.202301272018-0/s390x/rhcos-412.86.202301272018-0-metal.s390x.raw.gz",
+                "sha256": "66e823f693a5e5e404f9167774b65b227765119b20f2eae85a511d20fa85716e",
+                "uncompressed-sha256": "6b7b914483ffc38a887ccc84f388a4a6eaf91f5d0fd7cbda650324521477cb40"
               }
             }
           }
         },
         "openstack": {
-          "release": "412.86.202212081411-0",
+          "release": "412.86.202301272018-0",
           "formats": {
             "qcow2.gz": {
               "disk": {
-                "location": "https://rhcos.mirror.openshift.com/art/storage/prod/streams/4.12/builds/412.86.202212081411-0/s390x/rhcos-412.86.202212081411-0-openstack.s390x.qcow2.gz",
-                "sha256": "4029dfa4ebfee172c863ec982ef8614b081256418fa9885820364603286d455f",
-                "uncompressed-sha256": "aa52f831774ea0f7614a4167e986bf4d2a16958f2a75a1db67834a21e6574622"
+                "location": "https://rhcos.mirror.openshift.com/art/storage/prod/streams/4.12/builds/412.86.202301272018-0/s390x/rhcos-412.86.202301272018-0-openstack.s390x.qcow2.gz",
+                "sha256": "9ec6b6e2f0732eeea85cd1634e9b3b2f631ed05a9cea907b8bbc6c3e67944ddb",
+                "uncompressed-sha256": "0404efddb2f171cce925e02729acb75c2e19ddbd2dad5672edd1b5761dd8277b"
               }
             }
           }
         },
         "qemu": {
-          "release": "412.86.202212081411-0",
+          "release": "412.86.202301272018-0",
           "formats": {
             "qcow2.gz": {
               "disk": {
-                "location": "https://rhcos.mirror.openshift.com/art/storage/prod/streams/4.12/builds/412.86.202212081411-0/s390x/rhcos-412.86.202212081411-0-qemu.s390x.qcow2.gz",
-                "sha256": "ec09c39364d38a4a638eeafae9bdb5fd44e5f9e5b5056f9aab364374671ab71b",
-                "uncompressed-sha256": "4c76a419599c5c27beb712156df02ed9e5fae68ef280590d86b9424e38c73dfb"
+                "location": "https://rhcos.mirror.openshift.com/art/storage/prod/streams/4.12/builds/412.86.202301272018-0/s390x/rhcos-412.86.202301272018-0-qemu.s390x.qcow2.gz",
+                "sha256": "e275f5ee9fb9de60ab7cc439cf9550b1592b13855bd8cadd23f8c12f73d09d14",
+                "uncompressed-sha256": "6522abd03d4bb43919ea5de074db18c4edb25a64d3e3a8c3f3f2b347673e6e85"
+              }
+            }
+          }
+        },
+        "qemu-secex": {
```

Comment:
Just calling this out as a change, which I think was intended.

Reply:
Thanks Dusty! I updated the commit message but that didn't update the original comment.  This is one of the bugs that are fixed by this PR.